docs/SUMMARY.md

```diff
@@ -105,7 +105,6 @@
         * [offsetof](syntax_and_semantics/offsetof.md)
         * [Uninitialized variable declaration](syntax_and_semantics/declare_var.md)
     * [Compile-time flags](syntax_and_semantics/compile_time_flags.md)
```

Comment:
Compile time flags could also go under the "compilation" section

Reply:
The topic matches, but these are different categories of texts. *Compile-time flags* describes part of the language specification. The *Compilation* section is in the guides section and contains guides which expand on explaining practical use of specific topics.